pjsip/src/pjsua-lib/pjsua_im.c

```diff
@@ -304,13 +319,30 @@ static pj_bool_t im_on_rx_request(pjsip_rx_data *rdata)
         return PJ_TRUE;
     }
     
-    /* Respond with 200 first, so that remote doesn't retransmit in case
-     * the UI takes too long to process the message. 
-     */
-    pjsip_endpt_respond( pjsua_var.endpt, NULL, rdata, 200, NULL,
-                         NULL, NULL, NULL);
+    acc_id = pjsua_acc_find_for_incoming(rdata);
+    
+    if (acc_id != PJSUA_INVALID_ID && pjsua_var.acc[acc_id].cfg.auto_process_sip_message == PJ_FALSE
+            && is_typing_indication(rdata) == PJ_FALSE) {
```

Comment:
Minor:
- Coding style for [brace in multiline statement](https://docs.pjsip.org/en/latest/get-started/coding-style.html#use-k-r-style-brace-placement), the brace should be in a separate line.
- While in coding style, please also recheck for [line length limit](https://docs.pjsip.org/en/latest/get-started/coding-style.html#limit-line-length) in the whole PR code.